js/damage.js

```diff
@@ -543,6 +543,9 @@ function getDamageResult(attacker, defender, move, field) {
     if (attacker.ability === "Tinted Lens" && typeEffectiveness < 1) {
         finalMods.push(0x2000);
         description.attackerAbility = attacker.ability;
+    } else if (field.isFriendGuard) {
+        finalMods.push(0xC00);
+        description.attackerAbility = attacker.ability;
```

Comment:
`description.isFriendGuard = true;` and then deal with it where the others are in `buildDescription()`.

Your choice on what it actually says. "through Friend Guard"? "with an ally's Friend Guard"? "softened by Friend Guard"?